packages/angular/package.json

```diff
@@ -41,13 +40,15 @@
     "protractor": "~7.0.0",
     "ts-node": "~8.3.0",
     "tslint": "~6.1.0",
-    "typescript": "~4.1.5",
-    "xstate": "^4.20.1"
+    "typescript": "~4.1.5"
   },
   "workspaces": {
     "nohoist": [
       "@angular-devkit/build-angular"
     ]
   },
-  "typings": "./dist/ui-angular/index.d.ts"
+  "typings": "./dist/ui-angular/index.d.ts",
+  "publishConfig": {
+    "directory": "dist/ui-angular"
+  }
```

Comment:
Angular packager builds the package to publish in `dist/ui-angular` and not at its root. This is quite an anti-pattern to how most monorepo deployment tools publish, and you can find related conversation in https://github.com/lerna/lerna/issues/1282. 

Fortunately both lerna and changesets support subdirectory publish: https://github.com/atlassian/changesets/pull/428, though I haven't tested this out yet.

Reply:
An alternative that we can try is instead doing:

```json
"publishConfig": {
  "main": "dist/ui-angular/main.js",
  "module": "dist/ui-angular/module.js",
  ...
}
```

Reply:
Right, the problem is that this is not the `package.json` we want to pack for publish as mentioned in the comment above. We are interested in publishing `dist/ui-angular/package.json`, which has the library-specific dependencies.

Reply:
I can do a quick demo what the structure looks like, since this structure is pretty tricky.